execution/executor/src/lib.rs

```diff
@@ -682,14 +706,24 @@ impl<V: VMExecutor> BlockExecutor for Executor<V> {
                 &parent_block_executed_trees,
             );
 
+            // @SG -- In my opinion, it is at this point that we will split the VM functionality 
+            // and we will basically send the following in a channel: state_view and transactions.
+
+            let t1 = Instant::now();
+
             let vm_outputs = {
                 trace_code_block!("executor::execute_block", {"block", block_id});
                 let __timer = OP_COUNTERS.timer("vm_execute_block_time_s");
                 let _timer = LIBRA_EXECUTOR_VM_EXECUTE_BLOCK_SECONDS.start_timer();
                 V::execute_block(transactions.clone(), &state_view).map_err(anyhow::Error::from)?
             };
 
+            println!("VM TIME in executor/src/lib.rs: {:?}",t1.elapsed());
+
             trace_code_block!("executor::process_vm_outputs", {"block", block_id});
+
+            let t1 = Instant::now();
```

Comment:
Instant of using `Instant` + `println`, could you rewrite the timers using `trace_code_block!` (see line 723) or `observe_duration` as @andll suggested? This will let us see the timing in cluster-test.

Reply:
"Instant of using `Instant`" is good.... really good.... ;)

Reply:
LOL